CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+
+- The `@cumulus/ingest/util.lookupMimeType` function now returns `undefined`
```

Comment:
Opinion: this is a breaking change for sure, but I'm wondering if we could group all these code ones that probably 95% of users don't care about under a bullet like "Code changes"